get-involved/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+hero-image: url(../assets/images/stock-images/pie-staff.jpg)
+hero-message: Get Involved!
+---
+
+<!-- WHAT WE DO -->
+<div class="container">
+    <h1 style="text-align: center">Join PiE Staff</h1>
+    <!--<h2>WHAT WE DO</h2>-->
+</div>
+<div class="rc-row vertical-align" >
```

Comment:
I think this should also be wrapped in `<div class="container">` - it's currently taking up the full width of the page and looks a little jarring compared to everything else